.github/workflows/build-container.yml

```diff
@@ -0,0 +1,33 @@
+name: dispatched build-docker-image
+
+on:
```

Comment:
We also want to build this repo on pushes to master

Reply:
Right, makes sense!  Updated.